packages/typography/src/Typography.tsx

```diff
@@ -117,15 +100,19 @@ Subtitle.displayName = 'Subtitle';
 /**
  * Body
  */
-type BodyProps = HTMLElementProps<'div'> & {
+type BodyProps = HTMLElementProps<'p'> & {
```

Comment:
I'd probably keep `BodyProps` using a `div`, to mitigate any breaking style changes (`p` has built in margin that could break some layouts)

Reply:
Having `Body` be a `div` is interesting. It feels like we should make it a `p` and just remove the default margin? Otherwise it feels kinda weird semantically

Reply:
That's a good point. And I just noticed that we're using `sharedStyles` here, which will remove the default margin anyway